nautobot/core/tests/test_api.py

```diff
@@ -216,3 +216,62 @@ def test_header_and_query_version(self):
         response = self.client.get(f"{url}?api_version={min_version}", **self.header)
         self.assertHttpStatus(response, 400)
         self.assertIn("Version mismatch", response.data["detail"])
+
+
+class LookupTypeChoicesTestCase(APITestCase):
+    def test_get_lookup_choices_without_query_params(self):
+        url = reverse("lookup_choices")
+        response = self.client.get(url, **self.header)
+
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.data, "content_type and field_name are required parameters")
+
+    def test_get_lookup_field_data_invalid_query_params(self):
+        url = reverse("lookup_choices")
+        response = self.client.get(url + "?content_type=invalid.contenttypes&field_name=name", **self.header)
+
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.data, "content_type not found")
+
+    def test_get_lookup_choices(self):
+        url = reverse("lookup_choices")
+        response = self.client.get(url + "?content_type=dcim.site&field_name=status", **self.header)
```

Comment:
Do we need a test for invalid `field_name`, e.g. `?content_type=dcim.site&field_name=ice_cream_flavor`?